### DISCUSSION TITLE
Will there be more tiers?

### DISCUSSION THREAD

**Discussion by lucsoft** [fnuecke/oc2#13] [Q&A] [answered]
Currently in OpenComputers 1 there are 3 Tiers (+ Creative Tier) will there be any Tiers in OC2?

**Comment by lucsoft** [comment, marked as answer]
No, in OpenComputers2 we keep it by one tier only.

The only Items that will have tiers is Memory sticks and Hard drives

**Comment by atirutw** [comment]
I feel like this could have been in a FAQ page or something
